com.unity.shadergraph/package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "com.unity.shadergraph",
   "description": "The Shader Graph package adds a visual Shader editing tool to Unity. You can use this tool to create Shaders in a visual way instead of writing code. Specific render pipelines can implement specific graph features. Currently, both the High Definition Rendering Pipeline and the Universal Rendering Pipeline support Shader Graph.",
-  "version": "12.1.4",
+  "version": "12.1.5",
```

Comment:
it miss update of changelog of shadergraph

Reply:
thanks, fixed in next commit.